igvc_perception/src/train_eval/models/model.py

```diff
@@ -0,0 +1,71 @@
+import torch
```

Comment:
I would include a comment at the top of this file linking to the UNet paper. [link](https://arxiv.org/abs/1505.04597)

Reply:
done